src/main/java/seedu/address/model/person/RecordList.java

```diff
@@ -0,0 +1,38 @@
+package seedu.address.model.person;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Represents a record list in the address book.
+ *
+ */
+public class RecordList {
+    /* Record List variables */
+    public final List<Record> recordList;
```

Comment:
Might need to change these fields to private and create methods to access specific records in the list given index of the tasks. This is to prevent other classes from directly changing the list of records.